_specs/ecip-vanilla-sha-3.md

```diff
@@ -0,0 +1,73 @@
+```
+ECIP: TBD
+Title: Change the ETC Proof of Work Algorithm to _vanilla_ Sha-3
+Status: Draft
+Type: Network
```

Comment:
Can you make it Standards Track / Core?

Reply:
Update the status to Standards Track/ Core.